Sources/ZcashLightClientKit/Block/SaplingParameters/SaplingParametersHandler.swift

```diff
@@ -7,23 +7,37 @@
 
 import Foundation
 
-extension CompactBlockProcessor {
-    func handleSaplingParametersIfNeeded() async throws {
+struct SaplingParametersHandlerConfig {
+    let dataDB: URL
```

Comment:
this is named dataDb everywhere else. 
```suggestion
    let dataDb: URL
```